index.html

```diff
@@ -34,13 +35,42 @@ <h1>nav</h1>
               About us
             </a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <div class="dropdown-item__section">
-                <span class="dropdown-item--title">Overview</span>
+              <!-- @matt:
+
+              1. Title
+              title shouldn't be a Modifier of dropdown-item, because it doesn't modify dropdown-item.
+              "Title" describes it's purpose (meaning), not just it's appearance. So it should be an Element.
+
+              Definitions from https://en.bem.info/methodology/quick-start/
+              * Element: a "part of a block that can't be used separately from it... name describes its purpose"
+              * Modifier: "defines the appearance, state, or behavior... name describes it's appearance"
+
+              2. Section
+              What if dropdown is the block and...
+              * dropdown-item is an element of dropdown
+              * dropdown-section is an element of dropdown, not an element of dropdown-item
+              * dropdown-title is an element of dropdown, not a modifier of dropdown-item
+
+              All this fits with the Bootstrap BEM convention in my blog post
+              https://agility.zone/2020/09/11/bootstrap-bem/
+              -->
+              <div class="dropdown-section">
+                <span class="dropdown-title">Overview</span>
+                <!-- @matt:
+                Bootstrap has class="dropdown-header"
+                https://getbootstrap.com/docs/4.0/components/dropdowns/#menu-headers
+                -->
+
                 <a class="dropdown-item" href="#">What is BMJ Best Practice?</a>
               </div>
+              <!-- @matt:
```

Comment:
Not necessary to add extra markup, when we can achieve this with psuedo elements and selectors. The way I've wrote the scss allows us to add in sections and change links as / when needed and the design will stay the same. Section last child will have a border bottom but the last section won't have a border bottom - as per design.